modin/test/exchange/dataframe_protocol/base/test_sanity.py

```diff
@@ -0,0 +1,55 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Basic sanity checks for the DataFrame exchange protocol."""
+
+import pytest
+import modin.pandas as pd
+
+from modin.pandas.test.utils import default_to_pandas_ignore_string
+from modin.conftest import (  # noqa F401 (unused import, actually used in `test_basic_io`)
+    get_unique_base_execution,
+)
```

Comment:
what?.. just use it as a fixture - there should be no need to import, we might even stop shipping `modin.conftest` in the future

Reply:
Yes, we can remove it.

Reply:
removed